packages/react/src/components/Authenticator/SignUp/FormFields.tsx

```diff
@@ -99,7 +119,7 @@ export function FormFields() {
 
                 {validationError['confirm_password'] && (
                   <Text role="alert" variation="error">
-                    {translate(validationError['confirm_password'])}
+                    {translate(validationError['confirm_password'] as string)}
```

Comment:
Is this required because the formContext isn't properly typed?

Reply:
Same issue with `validationError`